safe_control_gym/controllers/lqr/ilqr.py

```diff
@@ -193,7 +196,7 @@ def update_policy(self, env):
 
         # Initialize backward pass.
         state_k = self.state_stack[-1]
-        input_k = env.U_GOAL
+        input_k = env.symbolic.U_EQ
```

Comment:
shouldnt all of these be self.model.U_EQ?

Reply:
I think so yes. This is similar to issue #93 and if we want to allow controllers to have access to the true linearization point or only the one provided by the prior. I changed them to `self.model.U_EQ` for now.